tests/common.cpp

```diff
@@ -103,6 +103,14 @@ result_t validateUInt16(__m128i a,
     return TEST_SUCCESS;
 }
 
+result_t validateInt32(__m64 a, int32_t u0, int32_t u1)
```

Comment:
Can you check line 38? There is already one `validateInt32`.

Reply:
But `validateInt32` at line 38 takes `__m128i` as input argument. The new `validateInt32` implemented here takes `__m64` as input argument.

Reply:
It was intended to add an overloading function. You shall mention in git commit message(s) as well.